src/bundle/Resources/public/scss/_dropdown.scss

```diff
@@ -273,6 +273,80 @@
         }
     }
 
+    &--selector {
+        .ibexa-dropdown {
+            &__selection-info {
+                height: calculateRem(24px);
+                padding-left: calculateRem(8px);
+                border-radius: calculateRem(12px);
```

Comment:
The sides of this dropdown are fully rounded so we need a border radius that equals half of its height. 50% won't work because this is not a square and we do not want to have an ellipsis.